clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -372,7 +367,7 @@ public static String formatAddress(String host, Integer port) {
      * Read a properties file from the given path
      * @param filename The path of the file to read
      */
-    public static Properties loadProps(String filename) throws IOException, FileNotFoundException {
+    public static Properties loadProps(String filename) throws IOException {
```

Comment:
I'd keep `FileNotFoundException` in the signature as it indicates a specific error as opposed to the more general IOException